encoders/image/BigTransferEncoder/README.md

```diff
@@ -1,3 +1,15 @@
 # BigTransferEncoder
 
-`BigTransferEncoder` is Big Transfer (BiT) presented by Google (https://github.com/google-research/big_transfer), this class use pretrained BiT to encode data from a ndarray, potentially B x (Channel x Height x Width) into a ndarray of `B x D`. Internally, :class:`BigTransferEncoder` wraps the models from https://storage.googleapis.com/bit_models/.
\ No newline at end of file
+`BigTransferEncoder` is Big Transfer (BiT) presented by Google (https://github.com/google-research/big_transfer), is a State-of-the-art transfer learning model for computer vision. This class uses pretrained BiT to encode data from an `ndarray`, potentially B x (Channel x Height x Width) into an `ndarray` of shape `B x D`. Internally, :class:`BigTransferEncoder` wraps the models from https://storage.googleapis.com/bit_models/.
```

Comment:
- Let's add link `Big Transfer (BiT)` to the URL instead of putting it in brackets
- Same with URL at end. Can use `here` instead